app/assets/stylesheets/tools.css.scss

```diff
@@ -83,4 +89,39 @@ ul.reproducibility_checklist {
     margin-right: 1em;
     display: inline;
   }
-}
\ No newline at end of file
+}
+
+// Pagination control
+.pagination {
+  margin-top: 1em;
+  margin-bottom: 2em;
+
+  a, span, em {
```

Comment:
Properties should be ordered display, float, padding<br>Each selector in a comma sequence should be on its own line